crates/astria-sequencer-relayer/src/main.rs

```diff
@@ -3,7 +3,6 @@ use std::process::ExitCode;
 use astria_eyre::eyre::WrapErr as _;
 use astria_sequencer_relayer::{
     metrics_init,
-    telemetry,
```

Comment:
Good catch. Is there a reexport in `lib.rs` that should also go?

Reply:
yes, removed!